packages/ember-flight-icons/CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ Improved resilience of SVG sprite loading script
 
 - @hashicorp/flight-icons@3.0.0
 
+_Since this is an update brand colors and product icons, we consider this a `major` version release_
```

Comment:
I'm not sure we need this note here – because we don't usually add such notes for each dependency bump it can mistakenly be considered a note for the `ember-flight-icons` and not for `flight-icons`